mapdb/src/main/java/com/adtsw/jdatalayer/mapdb/MapDBClient.java

```diff
@@ -26,16 +29,34 @@
     protected static Logger logger = LogManager.getLogger(MapDBClient.class);
     
     private final Map<String, DB> namespaces;
+    private final Map<String, String> baseStorageLocations;
```

Comment:
Are these supposed to hold namespace storage locations? If yes, can we rename accordingly?